GitUI/UserControls/FilterToolBar.cs

```diff
@@ -8,6 +8,7 @@ namespace GitUI.UserControls
 {
     internal partial class FilterToolBar : ToolStripEx
     {
+        internal const string ReflogButtonName = nameof(tsbShowReflog);
```

Comment:
should this be a readonly instead?
(internal is not exposed outside the assembly so I believe this is fine, public const is not)

Reply:
Internal consts are ok.